components/brave_wallet/browser/json_rpc_service.cc

```diff
@@ -1188,7 +1188,9 @@ void JsonRpcService::OnGetERC20TokenBalance(
     return;
   }
 
-  const auto& args = eth::DecodeEthCallResponse(*result, {"uint256"});
+  // (uint256)
+  auto type = eth_abi::Tuple().addTupleType(eth_abi::UintM(256)).build();
```

Comment:
`eth_abi::Uint()` or I'd rather add `Unit256()` which is the least ambiguous

Reply:
and same for two cases below

Reply:
I find `UintM(256)` a better choice here compared to `Uint()` since the actual type is `uint256`, so there's less confusion.

Also is defining a `Unit256()` really necessary? The idea of having `UintM(<size>)` was to have less code. Btw I renamed `UintM(<size>)` to `Uint(<size>)`.